Test/comp/ExternJavaString.dfy

```diff
@@ -3,8 +3,11 @@
 // In this example, the extern method obtains a Java string and returns it as such.
 // The Dafny code converts that Java string to a Dafny string.
 
-class {:extern "java.lang.String"} JavaString {
```

Comment:
Did this have to change? Why?

Reply:
Reverted this change. Also had to redo the entire implementation for it :-D